protos/bottle/account/v1/events.proto

```diff
@@ -9,6 +9,14 @@ message UserCreated {
   bottle.account.v1.User user = 1;
 }
 
+message UserDeleted {
+  bottle.account.v1.User user = 1;
+}
+
+message UserUpdated {
+  bottle.account.v1.User user = 1;
```

Comment:
This will probably need to be extended in the future. Off the top of my head, I can see some weird circular logic issues unless we track what service made the update. Maybe also include the fields that where actually changed :shrug:

Reply:
Sounds good. We can circle up tomorrow and discuss this more 🎉